tests/regression/roles/sonic_pki/defaults/main.yml

```diff
@@ -0,0 +1,100 @@
+---
+ansible_connection: httpapi
+module_name: tacacs_pki
+
+preparations_tests:
+  delete_pki:
+    - name: delete_all
+      description: Delete PKI
+      state: deleted
+      input:
+tests:
+  - name: test_case_01
+    description: Add new PKI configuration
+    state: merged
+    input:
+      security_profiles:
+        - profile_name: mysp
+        - profile_name: myspb
+          ocsp_responder_list:
+            - http://example.com/ocsp
+      trust_stores:
+        - name: myts
+          ca_name: CA2
+
+  - name: test_case_02
+    description: Update specific params of security-profile
+    state: merged
+    input:
+      security_profiles:
+        - profile_name: mysp
+          cdp_list:
+            - http://example.com/cdp
+          trust_store: myts
+          revocation_check: false
+  - name: test_case_03
```

Comment:
In addition to testing deletion of a list item, please expand this test case to cover deletion of other (non-list_ attributes (e.g. a security profile "trust_store").